build/common.sh

```diff
@@ -704,6 +704,7 @@ function kube::build::copy_output() {
     --filter='+ zz_generated.*' \
     --filter='+ generated.proto' \
     --filter='+ *.pb.go' \
+    --filter='+ types.go' \
```

Comment:
Have I every told you how people got my hopes up about how awesome bazel was, and then I found out it only worked on Linux?

Is the distributed/cached stuff for that ready yet?